makeflow/src/makeflow.c

```diff
@@ -1541,7 +1617,18 @@ int main(int argc, char *argv[])
 	if(batch_queue_type == BATCH_QUEUE_TYPE_LOCAL) {
 		explicit_remote_jobs_max = explicit_local_jobs_max;
 	}
-
+	if(!loc_info.local_disk)
```

Comment:
Due to the fact that you already initialize these values at line 1148, I don't think these will ever be executed. You will also want to set cores to load_average_get_cpus() initially as now it will be 1 by default.